docs/streams/developer-guide/kafka-streams-group-sh.html

```diff
@@ -0,0 +1,278 @@
+                        </ul>
+                    </li>
+                    <li><code>--reset-offsets</code>: Reset input-topic offsets (one group at a time; instances should be inactive). Choose exactly one specifier:
+                        <ul>
+                            <li><code>--to-earliest</code>, <code>--to-latest</code>, <code>--to-current</code>, <code>--to-offset &lt;n&gt;</code></li>
+                            <li><code>--by-duration &lt;PnDTnHnMnS&gt;</code>, <code>--to-datetime &lt;YYYY-MM-DDTHH:mm:SS.sss&gt;</code></li>
+                            <li><code>--shift-by &lt;n&gt;</code> (±), <code>--from-file</code> (CSV)</li>
+                        </ul>
+                        Scope:
+                        <ul>
+                            <li><code>--all-input-topics</code> or one/more <code>--topic &lt;name&gt;</code>; some builds also support <code>--all-topics</code> (all input topics per broker topology metadata).</li>
+                        </ul>
+                        Safety:
+                        <ul>
+                            <li>Requires <code>--dry-run</code> or <code>--execute</code>.</li>
+                        </ul>
+                    </li>
+                    <li><code>--delete-offsets</code>: Delete offsets for <code>--all-input-topics</code>, specific <code>--topic</code> names, or <code>--from-file</code>.</li>
+                    <li><code>--delete</code>: Delete Streams group metadata; optionally pass <code>--internal-topics &lt;list&gt;</code> to delete a subset of internal topics.</li>
+                </ul>
+            </div>
+
+            <div class="options-common">
+                <h3>Common flags</h3>
+                <ul>
+                    <li><code>--group &lt;id&gt;</code>: Target Streams group (application.id).</li>
+                    <li><code>--all-groups</code>: Operate on all groups (allowed with <code>--delete</code>).</li>
+                    <li><code>--bootstrap-server &lt;host:port&gt;</code>: Broker(s) to connect to (required).</li>
+                    <li><code>--command-config &lt;file&gt;</code>: Properties for AdminClient (security, timeouts, etc.).</li>
+                    <li><code>--timeout &lt;ms&gt;</code>: Wait time for group stabilization in some operations (default: 5000ms).</li>
+                    <li><code>--dry-run</code>, <code>--execute</code>: Preview vs apply for mutating operations.</li>
+                    <li><code>--help</code>, <code>--version</code>, <code>--verbose</code>: Usage, version, verbosity.</li>
+                </ul>
+            </div>
+        </div>
+
+        <div id="best-practices">
+            <span class="anchor" aria-hidden="true"></span>
+            <h2>Best practices and safety</h2>
+            <ul>
+                <li>Stop all application instances before resetting or deleting offsets to prevent immediate re‑joins with stale positions.</li>
```

Comment:
Do we allow to change offsets even if the group is not-empty? For the "classic" protocol this is not allowed, and I would have expected it's also not allowed for "consumer" and "streams" groups? -- It seem the behavior would be non-deterministic if we allow to commit offset while the group is non-empty?